src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -48,16 +54,32 @@ private static string[] GetBuiltinExtensionAssemblies()
 
         public Type[] GetStartupTypes()
```

Comment:
Is this used anywhere?

Reply:
Yes, This is called by the webjobs. When the host starts up it uses the instance of ScriptStartupTypeLocator provided by us and calls the GetStartupTypes() to locate the extensions.